spec/support/cache_store_helper.rb

```diff
@@ -9,11 +11,13 @@ def self.it_works_for_cache_backed_features(options)
         request.ip
       end
 
-      get "/", {}, "REMOTE_ADDR" => "1.2.3.4"
-      assert_equal 200, last_response.status
+      Timecop.freeze do
```

Comment:
Thoughts on having a helper test method instead called something like `within_same_period` or something similar, to capture the spirit of the freezing?

Reply:
there I made the changes. What do you think? I like it in some cases but in others where we freeze the time only to be able to generate the same key used by the cache it doesn't look so nice to me.

Also, any thoughts about dropping `Timecop` in favor of [ActiveSupport::Testing::TimeHelpers](https://api.rubyonrails.org/classes/ActiveSupport/Testing/TimeHelpers.html)?

Reply:
> Also, any thoughts about dropping `Timecop` in favor of [ActiveSupport::Testing::TimeHelpers](https://api.rubyonrails.org/classes/ActiveSupport/Testing/TimeHelpers.html)?

Not opposed to.